docker-compose.yml

```diff
@@ -52,3 +52,4 @@ services:
       # so we can reach Garden/Baggageclaim for debugging
       CONCOURSE_BIND_IP: 0.0.0.0
       CONCOURSE_BAGGAGECLAIM_BIND_IP: 0.0.0.0
+      CONCOURSE_GARDEN_DNS_PROXY_ENABLE: "true"
```

Comment:
I suspect this'll make it so `hack/overrides/containerd.yml` won't work - haven't tried this, but maybe we could change it to something like:

```diff
  services:
    worker:
      environment:
        CONCOURSE_RUNTIME: containerd
+       CONCOURSE_GARDEN_DNS_PROXY_ENABLE: ""
+       CONCOURSE_CONTAINERD_DNS_PROXY_ENABLE: "true"
```

Not sure if the "unsetting" of `CONCOURSE_GARDEN_DNS_PROXY_ENABLE` actually works, though

Reply:
Hadn't thought of that! I amended my commit to include this but use `~` which correctly unsets the env var (checked with `docker-compose exec` and also ran a job that needed DNS access on both containerd and guardian backends)

Reply:
Oh awesome, was not aware of ~!